web/src/main/java/com/bazaarvoice/emodb/web/settings/DatabusSuppressedEventConditionAdminTask.java

```diff
@@ -0,0 +1,42 @@
+package com.bazaarvoice.emodb.web.settings;
+
+import com.bazaarvoice.emodb.common.dropwizard.task.TaskRegistry;
+import com.bazaarvoice.emodb.databus.SuppressedEventCondition;
+import com.bazaarvoice.emodb.sor.condition.Condition;
+import com.bazaarvoice.emodb.sor.condition.Conditions;
+import com.bazaarvoice.emodb.sor.delta.deser.ParseException;
+import com.google.inject.Inject;
+
+/**
+ * Task for managing the condition used to suppress databus events when a subscription opts to ignore suppressed events.
```

Comment:
Should we call it a "suppressed" event condition? Or simply a default "augmented" condition. Reason is it may get confusing dealing with double negatives. Making it more generic may take the confusion away.